Formula/mutanus.rb

```diff
@@ -0,0 +1,22 @@
+class Mutanus < Formula
+  desc "Performs mutation testing of you Swift project"
+  homepage "https://github.com/SoriUR/mutanus/"
+  url "https://github.com/SoriUR/mutanus/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "73204fe4b1788c2427d2d12f8a5d2579469cd8910ecfa777974d431118d83155"
+  license "MIT"
+
+  depends_on xcode: [">= 12.5", :build]
+  depends_on macos: [
+    :catalina,
+    :big_sur,
+  ]
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    bin.install ".build/release/mutanus"
+  end
+
+  test do
+    system "#{bin}/mutanus", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I didn't manage to write another test. This tool requires a swift project to run properly and has no other commands.
I can potentially create a command just for this test. Or I can try to create test project, download it, and execute the test

Reply:
The latter seems good.

Reply:
I've tried this
```
class Mutanus < Formula
  desc "Performs mutation testing of you Swift project"
  homepage "https://github.com/SoriUR/mutanus/"
  url "https://github.com/SoriUR/mutanus/archive/refs/tags/v0.1.1.tar.gz"
  sha256 "a534071a0f6f234cd3dc412afab6e825d54181fd8de0a8cc2a3478f1ba2189a8"
  license "MIT"

  depends_on xcode: [">= 12.5", :build]
  depends_on macos: :catalina

  resource("testdata") do
    url "https://github.com/SoriUR/MutanusTest/archive/refs/tags/v1.0.1.zip"
    sha256 "4680528a2697ca2a9f956b2c2756d29bb350ceabf1ec208cdef4ff5fce271cb9"
  end

  def install
    system "swift", "build", "--disable-sandbox", "-c", "release"
    bin.install ".build/release/mutanus"
  end

  test do
      # download test project    
    resource("testdata").stage do
      # download simulator
      system "xcrun simctl create \"iPhone\" \"iPhone 11 Pro Max\" iOS14.5"  
      # building test project
      system "xcodebuild test -project MutanusTest.xcodeproj -scheme MutanusTest -destination \"platform=iOS Simulator,name=iPhone 12 Pro Max\""

    end
  end
end
```

but got error
```
Testing failed:
	Couldn't create workspace arena folder '/Users/DD/MutanusTest-gyxuazfdcgumyheydlytxkovhuyu': You don’t have permission to save the file
```
	
Do you have any ideas?